#Service to clear the joint waypoint buffer
---
